aldryn_apphooks_config/widgets.py

```diff
@@ -15,21 +17,24 @@ def render(self, name, value, attrs=None, choices=()):
             out = super(AppHookConfigWidget, self).render(name, value, attrs, choices)
         else:
             out = super(AppHookConfigWidget, self).render(name, value, attrs)
-        final_attrs = self.build_attrs(attrs, name=name)
-        script = """
-        <script>
-        (function($) {
-            $(document).ready(function () {
-                var sel = $('#%(id)s');
-                if (!sel.val()) {
-                    sel.change(function () {
-                        $(this).apphook_reload_admin(
-                            '%(name)s', $("option:selected",$(this)).val()
-                        );
-                    });
-                }
-            });
-        })(django.jQuery);
-        </script>
-""" % final_attrs
-        return mark_safe(script + out)
+        if not DJANGO_1_10:
```

Comment:
I suggest to render the template manually if django < 1.10, this would avoid having the same js in two separate places.

Reply:
I haven't thought about that. That's a nice approach